app/views/notes/test.html.erb

```diff
@@ -0,0 +1 @@
+<input id="test">
```

Comment:
Useful file?

Reply:
no, I use to test my code before implementation